services/QuillLMS/engines/evidence/lib/evidence/evidence/check.rb

```diff
@@ -23,6 +23,8 @@ class NoMatchedFeedbackTypesError < StandardError; end
     }
 
     def self.get_feedback(entry, prompt, previous_feedback, feedback_types=nil)
+      entry = normalize_entry_text(entry)
```

Comment:
let's assign this to a new variable rather that overwriting `entry`.

Reply:
Good call.  Updating.